tidb-cloud/changefeed-sink-to-apache-kafka.md

```diff
@@ -47,10 +47,11 @@ For example, if your Kafka cluster is in Confluent Cloud, you can see [Resources
 
```

Comment:
this might conflict with https://github.com/pingcap/docs/pull/11515/files#diff-c95401c1c761cca89172f4f336be2ef3421dd1c9d83e635d8fb8d8fae3372b35